src/main/java/se/bjurr/gitchangelog/internal/git/GitRepo.java

```diff
@@ -291,6 +366,10 @@ private String getTagName(final Map<String, Ref> tagPerCommitHash, final String
     final RevCommit from = this.revWalk.lookupCommit(fromObjectId);
     final RevCommit to = this.revWalk.lookupCommit(toObjectId);
 
+    if (from == to) {
+      return newArrayList();
```

Comment:
If there is a tag on `from` then this list should not be empty.

Reply:
Generally speaking - yes, but it looks like we are only interested in the diff between the two commits in this case?
If the tag of a single `from` commit, which is also the `to` commit, is returned, we get a non-empty diff, just like in the #62 description.